inc/core/patterns/bcgov-alpha3-3x3-questions-answers.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Alpha V3 3x3 Questions and Answers.
+ *
+ * @since 1.7.0
+ *
+ * @package Bcgov/Theme/Block
+ */
+
+ namespace Bcgov\Theme\Block;
+
+return [
+	'title'      => __( '3x3 Questions and Answers', 'bcgov_blocks_theme' ),
+	'categories' => [ 'bcgov-blocks-theme-general', 'bcgov-blocks-theme-alpha-v3' ],
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"margin":{"top":"6rem","bottom":"6rem"},"padding":{"right":"2rem","left":"2rem"}}},"layout":{"type":"constrained"},"metadata":{"name":"Questions and Answers"}} -->
+	<div class="wp-block-group alignfull" style="margin-top:6rem;margin-bottom:6rem;padding-right:2rem;padding-left:2rem"><!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+	<div class="wp-block-group alignwide"><!-- wp:heading {"level":3,"style":{"elements":{"link":{"color":{"text":"var:preset|color|primary-brand"}}}},"textColor":"primary-brand","fontSize":"extra-large"} -->
+	<h3 class="wp-block-heading has-primary-brand-color has-text-color has-link-color has-extra-large-font-size">Your questions, answered.</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:columns {"className":"faq-columns"} -->
```

Comment:
Are the .faq-columns, .faq, etc. classes for potential future use? I'm unable to locate and specific styles attached to them.

Reply:
This has been fixed, classes removed.